### PR TITLE
chore: reintroduced well classes

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

Some apps still use .well bootstrap classes. In order to fix the regression we reintroduce these classes. We'll eventually replace them in future in favour of component
## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

**Review by domhorvathtw — APPROVED** [review]

**Comment by anttio** [comment]
Should this also be part of the Neptune bundle, or just a standalone CSS component?

File: packages/css/src/less/wells.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// TODO: remove bootstrap dependencies
+
+@import (reference) 'bootstrap/less/variables.less';
+@import 'bootstrap/less/wells.less';
```
Comment (anttio) [inline packages/css/src/less/wells.less:4]: Should we just refactor and clean this now? The implementation with mixins in our Bootstrap is just creating different size variants with same values as the default one. 
https://github.com/transferwise/bootstrap/blob/59d55b847efb4054173214670e8ea345176d70f1/dist/css/bootstrap.css#L10273

Comment (anttio) [reply packages/css/src/less/wells.less:4]: > Control on which breakpoints well styles are applied with some responsive classes. Works with "xs" "sm" "md" "lg" "xl"

Actually, seems that these might be needed. So nevermind this refactoring.

Comment (andreapiras) [reply packages/css/src/less/wells.less:4]: Ok do you want me to copy the generated css?

**Review by anttio — APPROVED** [review]

**Pull request merged**